packages/prompts/test/__snapshots__/autocomplete.test.ts.snap

```diff
@@ -12,7 +12,7 @@ exports[`autocomplete > can be aborted by a signal 1`] = `
 [36m│[39m  [2m○[22m [2mCherry[22m
 [36m│[39m  [2m○[22m [2mGrape[22m
 [36m│[39m  [2m○[22m [2mOrange[22m
-[36m│[39m  [2m[2m↑/↓[22m[2m to select • [2mEnter:[22m[2m confirm • [2mType:[22m[2m to search[22m
+[36m│[39m  [2m[2m↑/↓[22m to select • [2mEnter:[22m confirm • [2mType:[22m to search[22m
```

Comment:
this shows that `styleText` doesn't handle nested colours. e.g.:

```ts
styleText(`i am dim ${styleText('bold', 'i am bold')} i am no longer dim`);
```

we should search our code for any nested styles and double check they actually work